src/thor/astar.cc

```diff
@@ -162,7 +162,8 @@ void AStarPathAlgorithm::ExpandForward(GraphReader& graphreader,
 
     // Compute the cost to the end of this edge
     auto edge_cost = costing_->EdgeCost(directededge, tile);
-    Cost newcost = pred.cost() + edge_cost + costing_->TransitionCost(directededge, nodeinfo, pred);
+    auto transition_cost = costing_->TransitionCost(directededge, nodeinfo, pred);
```

Comment:
lots of changes like this in all the algorithms where i keep the transition cost separate so i can store it on the label when its updated or added